schema/settings.json

```diff
@@ -0,0 +1,137 @@
+{
```

Comment:
Nice!

Reply:
(yeah, this one is easy to generate, workbooks not so much)